src/main/java/creativei/service/impl/FollowUpServiceImpl.java

```diff
@@ -22,6 +22,16 @@ public FollowUp createFollowUp(FollowUp followUp) throws InvalidParamRequest {
 
         return followUpDao.save(followUp);
     }
+
+    @Override
+    public FollowUp updateFollowUp(FollowUp followUp) throws InvalidParamRequest {
+        if (!validateFollowUp(followUp))
+            throw new InvalidParamRequest("Required field can not be empty");
+
+        return followUpDao.save(followUp);
+
+    }
+
     private Boolean validateFollowUp(FollowUp followUp) {
         if (followUp.getCaseIndex() == null || followUp.getFollowUpDate() == null || followUp.getType() == null || followUp.getStatus() == null || followUp.getSubStatus() == null) {
```

Comment:
This is too generic. logger.error for each invalid scenario. If any required field is null, it should be logged. Add logging to other api validation methods as well

Reply:
This should be fixed.

Reply:
logger.error added is commit 3f3804c

Reply:
issue solved in commit bc17d78. Please review

Reply:
Logger messages are updated in commit 1229e82